apps/omg_watcher/lib/omg_watcher/block_getter/supervisor.ex

```diff
@@ -30,6 +30,7 @@ defmodule OMG.Watcher.BlockGetter.Supervisor do
     # such a situation will cause error when trying to execute already executed transaction
     children = [
       {OMG.State, []},
+      {OMG.Watcher.BlockGetter.Status, []},
```

Comment:
I was hesitant whether this is better, or just a direct `start_link` within `OMG.Watcher.BlockGetter`'s init, to make them even more tightly bound

Reply:
yep, that would make more sense. For what it's doing, an `Agent` would suffice as well.

Reply:
ok, will change that.

I was considering `Agent` but went with `GenServer` only for the sake of limiting the toolset currently employed. But if you're making this point I'll go with `Agent`